index2.html

```diff
@@ -277,13 +275,15 @@ <h2 style="font-size:1.25rem;">Analyze plant health. <br> Visualize photosynthes
           <div class="card text-center sample-images">
             <div class="card-body">
               <p class="card-text" style="font-size:.625rem;"><small>Drop your media here, connect through your computer above <br class="sample-text-effect"> or <br> explore with our ready-to-use images</small></p>
-              <div class="d-flex justify-content-around">
-                <div class="d-flex flex-column justify-content-center align-items-baseline">
-                  <img src="assets/red-filtered-trees-thumbnail.jpeg" alt="Red filtered image - select to explore the tool." style="width:5rem;height:auto;border:.5rem solild var(--image-container-bg);">
-                  <p class="card-text" style="font-size:.625rem;padding-top:.5rem;"><small>Red filtered trees</small></p>
+              <div id="sample-image__select" class="d-flex justify-content-around">
+                <div id="red-filtered-img" class="rfi d-flex flex-column justify-content-center align-items-baseline">
+                  <img class="rfi" src="assets/red-filtered-trees-thumbnail.jpeg" alt="Red filtered image - select to explore the tool." style="width:5rem;height:auto;border:.5rem solild var(--image-container-bg);">
```

Comment:
```suggestion
                  <img class="sample-image red-filter" src="assets/red-filtered-trees-thumbnail.jpeg" alt="Red filtered image - select to explore the tool." style="width:5rem;height:auto;border:.5rem solild var(--image-container-bg);">
```

I explain this idea below - and as to naming, I really like longer descriptive names because it often means we don't have to use as many comments!